glslc/src/main.cc

```diff
@@ -116,6 +119,16 @@ int main(int argc, char** argv) {
         return 1;
       }
       compiler.SetOutputFileName(file_name);
+    } else if (arg.starts_with("-working-directory")) {
+      std::string option = "-working-directory";
+      if (arg[option.size()] == '=') option = "-working-directory=";
```

Comment:
Just tried with clang, and this is, indeed, correct (see below).  Maybe we should put a comment in, for the future reader.

The following commands all all work:
    clang++ -working-directory=.  /tmp/x.cc 
    clang++ -working-directory.  /tmp/x.cc 
    clang++ -working-directory .  /tmp/x.cc

But with -std:
    deki>clang++ -std=c++11 /tmp/x.cc 
    deki>clang++ -stdc++11 /tmp/x.cc 
    clang: error: unknown argument: '-stdc++11'
    deki>clang++ -std c++11 /tmp/x.cc 
    clang: error: unknown argument: '-std'
    clang: error: no such file or directory: 'c++11'